### PR TITLE
Scalafmt - pure refactoring

### PR DESCRIPTION
**Pull request by mccartney**
Applying `scalafmt` from `sbt`

### CODE REVIEW

**Comment by mccartney** [comment]
I wonder why it hasn't been done earlier. Or what has changed recently.

**Comment by mwz** [comment]
I'm sure the codebase was formatted previously, but it looks like there's been a bunch of changes in 2.5, so perhaps some configuration has changed which forces it to be reformatted again.

**Comment by mccartney** [comment]
@mwz, would you want to double check or should I proceed with merging?

I am fine with either formatting styles. Just trying to limit down the changes, esp. that the formatting happens by itself.

File: src/main/scala/com/sksamuel/scapegoat/Feedback.scala [hunk 1]
```diff
@@ -60,7 +60,7 @@ class Feedback(
     if (shouldPrint(warning)) {
       val snippetText = snippet.fold("")("\n  " + _ + "\n")
       val report = s"""|[scapegoat] $text
-                       |  $explanation$snippetText""".stripMargin
+                      |  $explanation$snippetText""".stripMargin
```
Comment (mwz) [inline src/main/scala/com/sksamuel/scapegoat/Feedback.scala:63]: Seems ok to me except for this, which doesn't look right. I'm wondering what happened there.

Comment (mccartney) [reply src/main/scala/com/sksamuel/scapegoat/Feedback.scala:63]: Why would we care? Indeed it looks weird why would `scalafmt` perform a change like this.

But even with that change the output is the same, right?

**Comment by mccartney** [comment]
Ah. Not sure if it's because of #361, but I do have changes after running `scalafmt` from latest master again.

**Pull request merged**